src/components/grid-aware/TwoParagraphBlock/TwoParagraphBlock.module.css

```diff
@@ -0,0 +1,143 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  grid-template-rows: auto 0;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedBackground {
+  background: var(--color-gray-300);
+  grid-column: 1 / span 2;
+  grid-row: 1 / -1;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr;
+  }
+
+  .bleedBackground {
+    grid-column: 1 / span 3;
+  }
+}
+
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  grid-template-rows: auto 590px;
+  padding-top: 122px;
+  row-gap: 84px;
+}
+
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding-bottom: 80px;
+    padding-top: 50px;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaTopLeft {
+  grid-column: 2 / span 4;
+  grid-row: 1;
+}
+
+.gridAreaTopRight {
+  grid-column: 6 / span 6;
+  grid-row: 1;
+}
+
+.gridAreaBottomLeft {
+  grid-column: 2 / span 5;
+  grid-row: 2;
+}
+
+.gridAreaBottomRight {
+  grid-column: 8 / span 4;
+  grid-row: 2;
+}
+
+
+/* Component wrappers */
+
+.paragraph1Wrapper {
+  font: var(--font-body-large);
+  margin-top: 0;
+}
+
+.paragraph2Wrapper {
+  font: var(--font-body-small);
+  margin-top: 0;
+}
+
+.image1Wrapper {
+  position: relative;
+  top: 35px;
+}
+
+@media (--tablet-and-down) {
+  .paragraph1Wrapper {
+    margin-right: 40px;
+    margin-top: 40px;
+  }
+
+  .paragraph2Wrapper {
+    margin-right: 40px;
+    margin-top: 40px;
+  }
+
+  .image1Wrapper {
+    margin-left: 40px;
+    margin-top: 40px;
+    position: static;
```

Comment:
Although position static does the behavior wanted. position: inherit is should be more appropriate right @richardxia ? We don't want to introduce new "un-needed" positions to elements right?

Reply:
I don't think you ever want to use `position: inherit`. `position` is not a property that is normally inherited, so it's really strange to force it to be inherited. `inherit` means "set this to whatever the parent's value for this is". If the parent has `position: absolute`, then this would also get `position: absolute`; if the parent has `position: fixed`, then this would also get `position: fixed`. I can't think of a single situation where you would ever want this kind of a behavior.

`position: static` will reset this back to the normal behavior for all elements (_all_ elements start with `position: static`), so I think this is fine.

You could also invert the direction of the media query and wrap the other selector with `@media (--laptop-and-up)`, and that way, you wouldn't have to reset this property.